File: thru-hiker-log-app/src/app/models/entry.ts
export interface IEntry {
  id?: number;
  trailId: number;
  name: string;
  startTime: Date;
  endTime: Date;
  startMileage: number;
  endMileage: number;
  elevationGain?: number;
  elevationLoss?: number;
  weather?: string;
  temperature?: number;
  notes?: string;
  dayNumber?: number; // computed field, not stored in DB
}

File: ThruHikerLogApi/ThruHikerLogApi/Models/Entry.cs
namespace ThruHikerLogApi.Models
{
    public class Entry
    {
        public int Id { get; set; }
        public int TrailId { get; set;  }
        public string Name { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public double StartMileage { get; set; }
        public double EndMileage { get; set; }
        public string Notes { get; set; }
    }
}
